### PR TITLE
Create a non-technical request template

### PR DESCRIPTION
**Pull request by hangfei**
create a non-technical request template for non-technical issues like documentation improvements etc.

## Description

Resolves #XXX

## Does this PR introduce any user-facing changes?

- [x] No. You can skip the rest of this section.
- [ ] Yes. Make sure to clarify your proposed changes.

### CODE REVIEW

**Review by xiaoyongzhu — DISMISSED** [review]

**Comment by hangfei** [comment]
@ahlag  Could you help review it as well since you create all other templates and you are most knowledgeable about this? Thanks!

**Comment by ahlag** [comment]
@hangfei 
Yeah sure.
By the way, I do not have review privilege.
screen_shot 2022-08-19 at 12 13 59

File: .github/ISSUE_TEMPLATE/non_technical_request_template.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name: Non-technical Request
+description: Use this template for non-technical requests
+labels: 'enhancement'
```
Comment (ahlag) [inline .github/ISSUE_TEMPLATE/non_technical_request_template.yaml:3]: How about we add a `non-technical` tag?
```
labels: ['enhancement', 'non-technical']
```

**Comment by ahlag** [comment]
@hangfei @xiaoyongzhu 
How does the Feathr team want to categorize the labels of the issues? 
We can relate labels to specific templates.

File: .github/ISSUE_TEMPLATE/non_technical_request_template.yaml (hunk 1 above)
Comment (hangfei) [reply .github/ISSUE_TEMPLATE/non_technical_request_template.yaml]: good point. addressed.

**Comment by hangfei** [comment]
> 

Right now, labels are helpers mostly. So i think we can attach labels based on different scenarios. There is no strong mapping needed between labels and template.

**Comment by ahlag** [comment]
@hangfei 
Btw, do you think we need a doc_fix_template like MLFlow? Right now we only have Feature Request and Bug Fix templates.
https://github.com/mlflow/mlflow/blob/master/.github/ISSUE_TEMPLATE/doc_fix_template.yaml

**Review by xiaoyongzhu — APPROVED** [review]

**Review by blrchen — APPROVED** [review]

**Pull request merged**